android/app/src/main/java/net/pengcook/android/presentation/core/util/ImageUtils.kt

```diff
@@ -0,0 +1,78 @@
+package net.pengcook.android.presentation.core.util
+
+import android.content.Context
+import android.content.pm.PackageManager
+import android.net.Uri
+import androidx.core.content.ContextCompat
+import androidx.core.content.FileProvider
+import java.io.File
+import java.io.IOException
+import java.text.SimpleDateFormat
+import java.util.Date
+import java.util.Locale
+
+object ImageUtils {
+    private fun createTempImageFile(context: Context): File {
```

Comment:
ImageUtil 생성자로 Context를 넣는 것은 어떠실까요?
내부에 메소드들이 모두 context를 공통적으로 받고 있는데, 사용하는 곳에서 context를 넣어놓은 객체를 선언해놓고 사용하면 좋을 것 같습니다!
```suggestion
class ImageUtils(private val context: Context) {
    private fun createTempImageFile(): File {
```